plugwise_usb/nodes/__init__.py

```diff
@@ -423,16 +428,29 @@ async def node_info_update(
     ) -> bool:
```

Comment:
To be consistent the node_info_update function (just like the other *_update functions) should return the populated `NodeInfo `dataclass object or None if it failed.